core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -286,11 +286,12 @@ <h3>Roles</h3>
                   </span>
                 </div>
 
-                <input type="submit" class="btn btn-default protractor-test-save-role" value="Save">
+                <input type="submit" class="btn btn-default protractor-test-save-role" value="Save" ng-disabled="hasTitleNotBeenGiven()">
```

Comment:
Make sure to test your changes before submitting. This function here doesn't exist anymore.

Reply:
Im sorry, I did not see it. (I thought clicking on "commit changes button" will change all of it, it didnt for 1 so I changed that, Ill correct everything and push changes rn)

Reply:
rechecked and pushed now, 
Thanks!